pkg/services/accesscontrol/models.go

```diff
@@ -285,32 +285,31 @@ const (
 	ActionAPIKeyDelete = "apikeys:delete"
 
 	// Users actions
-	ActionUsersRead     = "users:read"
-	ActionUsersWrite    = "users:write"
-	ActionUsersTeamRead = "users.teams:read"
+	ActionUsersRead  = "users:read"
+	ActionUsersWrite = "users:write"
 	// We can ignore gosec G101 since this does not contain any credentials.
 	// nolint:gosec
-	ActionUsersAuthTokenList = "users.authtoken:list"
+	ActionUsersAuthTokenList = "users.authtoken:read"
```

Comment:
nit: Change constant to ActionUsersAuthTokenRead

Reply:
Do you mind if I change the constant names in a separate PR? This one is pretty big already, and I don't want it to grow even huger. I promise I'll actually do it 😄

Reply:
Yeah i think that would be better :)